src/openforms/submissions/tasks.py

```diff
@@ -0,0 +1,232 @@
+import logging
+from datetime import timedelta
+
+from django.db.models import (
+    Case,
+    CharField,
+    DateTimeField,
+    DurationField,
+    ExpressionWrapper,
+    F,
+    IntegerField,
+    Value,
+    When,
+)
+from django.utils import timezone
+
+from ..celery import app
+from ..config.models import GlobalConfiguration
+from ..submissions.constants import RemovalMethods
+from .constants import RegistrationStatuses
+from .models import Submission
+
+logger = logging.getLogger(__name__)
+
+
+@app.task
+def delete_submissions():
```

Comment:
Please add `logger.info(...)` calls so we have some feedback about what happend. I need to see how many objects were deleted/updated for every operation.